src/work-allocation/containers/my-tasks/my-tasks.component.spec.ts

```diff
@@ -75,14 +75,19 @@ describe('MyTasksComponent', () => {
     expect(component.tasks.length).toEqual(2);
   });
 
-  it('should have all columns, including "Manage +"', () => {
+  it('should have all column headers, including "Manage +"', () => {
     const element = fixture.debugElement.nativeElement;
     const headerCells = element.querySelectorAll('.govuk-table__header');
     const fields = component.fields;
     expect(headerCells).toBeDefined();
     expect(headerCells.length).toEqual(fields.length + 1); // Extra one for Manage +;
     for (let i = 0; i < fields.length; i++) {
-      expect(headerCells[i].textContent).toEqual(fields[i].columnLabel);
+      // ensure derivedIcon has no header and every other field does
+      if (fields[i].name !== 'derivedIcon') {
```

Comment:
I think a better condition would be `if (fields[i].columnLabel)` as it's more generic.

Reply:
Done